### ISSUE TITLE
Some symbols not displaying correctly, letters in math script and "\in" conceal for example

### ISSUE THREAD

**Issue by isakvobj** [KeitaNakamura/tex-conceal.vim#41]
Hi, I am not that integrated into neovim, but I want to get a working LaTeX environment for writing math papers and taking notes. Your plugin caught my I attention, and I read up on others using it for writing with VimTeX so I tried out your plugin. For the most part it seems to work, but some symbols are not displaying correctly. The math script symbols are not that important, and I can always change the source for my tex-conceal download to remove that functionality, but when looking through the source code of the plugin I did not find out what conceals \in. I troubleshooted a bit, made sure that it did not work even when explicitly setting the decoding to utf-8, and made sure I used the recommended configurations, but it still does not work. I tried removing VimTeX, but that did not solve it. I even tried removing tex-conceal, and it seemed like the config for conceallevel still operated as normal. 

This could indicate that the issue is not with this plugin, but I still hope that you could have some insight into what could cause the issue. If you know how I could disable the concealing for \in or other conceal matches that would also be of huge help. I am using NVIM v0.9.1 on a Windows 10 laptop. Below are visuals for the issues I experience, as well as my init.vim file.

Test tex file, plus 2 extra lines to further demonstrate:
![image](https://github.com/KeitaNakamura/tex-conceal.vim/assets/140022043/d0cb3483-70d2-4f11-8dcd-515787632e09)

Test tex file highlighted:
![image](https://github.com/KeitaNakamura/tex-conceal.vim/assets/140022043/69b69ecf-69d0-467e-a128-599cd9953093)

init.vim file:
![image](https://github.com/KeitaNakamura/tex-conceal.vim/assets/140022043/28307b3a-b9a3-4a21-b64d-af94b09ea762)

**Comment by KeitaNakamura** [comment]
Hi. [vimtex](https://github.com/lervag/vimtex) actually already has excellent support for the conceal feature. I think you can just use vimtex alone if there is no specific reason. The missing characters in your screenshots are likely because the font you are using doesn't have those characters.

**Comment by isakvobj** [comment]
Ok, thanks for the reply. In trying to troubleshoot this, I also noticed that I had no app icon when running the application, and checked if I used the correct app. I saw I had Neovim Qt which had the correct icon, and tried to run that instead. That worked, and by changing the highlight conceal configs I got my desired setup. I think the app i ran previously was Neovim.exe, not sure if you are supposed to run it, but in any case it had no icon, and it seems like it messed up some of the functionality.

Based on my limited understanding of neovim I now think that you are supposed to run Neovim.exe from a separate GUI, and that is why it is now working and why neovim.exe had no icon, because you are not supposed to use it as an app. Sorry for any confusion my lack of knowledge may have caused.
